apps/aechannel/test/aesc_fsm_SUITE.erl

```diff
@@ -183,45 +214,101 @@ init_per_group(state_hash, Config0) ->
+                 {bench_rounds, 100} %% a lower amount than the default 1000
+                    | Config1],
+            case Group of
+                initiator_is_ga ->
+                    attach_initiator(Config2),
+                    initiator_spend(Config2);
+                responder_is_ga ->
+                    attach_responder(Config2),
+                    responder_spend(Config2);
+                both_are_ga ->
+                    attach_initiator(Config2),
+                    initiator_spend(Config2),
+                    attach_responder(Config2),
+                    responder_spend(Config2)
+            end,
+            Config2
+    end;
+init_per_group(generalized_accounts, Config) ->
+    Config;
 init_per_group(_Group, Config) ->
     init_per_group_(Config).
 
 init_per_group_(Config) ->
-    aecore_suite_utils:start_node(dev1, Config),
-    aecore_suite_utils:connect(aecore_suite_utils:node_name(dev1)),
-    ct:log("dev1 connected", []),
-    try begin
-            Initiator = prep_initiator(dev1),
-            Responder = prep_responder(Initiator, dev1),
-            [{initiator, Initiator},
-             {responder, Responder},
-             {port, ?PORT},
-             {initiator_amount, 1000000 * aec_test_utils:min_gas_price()},
-             {responder_amount, 1000000 * aec_test_utils:min_gas_price()}
-             | Config]
-        end
-    catch
-        error:Reason ->
-            Trace = erlang:get_stacktrace(),
-            catch stop_node(dev1, Config),
-            error(Reason, Trace)
+    case proplists:get_value(ga_group, Config, false) of
+        true -> Config;
+        false ->
+            aecore_suite_utils:start_node(dev1, Config),
+            aecore_suite_utils:connect(aecore_suite_utils:node_name(dev1)),
+            ct:log("dev1 connected", []),
+            try begin
+                    Initiator = prep_initiator(dev1),
+                    Responder = prep_responder(Initiator, dev1),
+                    [{initiator, Initiator},
+                    {responder, Responder},
+                    {port, ?PORT},
+                    {initiator_amount, 10000000 * aec_test_utils:min_gas_price()},
+                    {responder_amount, 10000000 * aec_test_utils:min_gas_price()}
+                    | Config]
+                end
+            catch
+                error:Reason ->
+                    Trace = erlang:get_stacktrace(),
+                    catch stop_node(dev1, Config),
+                    error(Reason, Trace)
+            end
     end.
 
-end_per_group(_Group, Config) ->
-    _Config1 = stop_node(dev1, Config),
+end_per_group(Group, Config) ->
+    case proplists:get_value(ga_group, Config, false) of
+        true ->
+            case lists:member(Group, [initiator_is_ga,
+                                      responder_is_ga, both_are_ga]) of
+                true ->
+                    _Config1 = stop_node(dev1, Config);
+                false -> pass
```

Comment:
When does this happen?

Reply:
This might be a bit too implicit: the groups that belong to `generalized_accounts` group - namely - `initator_is_ga`, `responder_is_ga` and `both_are_ga` start with setting up one or two generalized accounts. Usually nodes are being stopped at each group end but since we can save some contracts upgrading: we don't stop the node when finishing any of the following groups:
```
  ga_sequence() ->
      [ {group, transactions}  
      , {group, errors}
      , {group, signatures}
      , {group, channel_ids}                                 
      ].
```

To answer your question: while finishing `transactions` group for example:
* when being ran outside of the scope of `generalized_accounts` group - there is no `ga_group` being set, so we stop the node (to start it with the next group)
* when being ran in the context of any of the  `generalized_accounts` groups - we do not stop the node (so we don't lose the GA setup). Instead we stop it once finishing the group